tests/test_cargo_compile.rs

```diff
@@ -864,6 +864,11 @@ Caused by:
 });
 
 test!(lto_build {
+    // FIXME: currently this hits a linker bug on 32-bit MSVC
+    if cfg!(all(target_env = "msvc", target_pointer_width = "32")) {
+        return
+    }
```

Comment:
Note that this is connected to https://github.com/rust-lang/rust/pull/27224, so once that lands and makes its way into the nightlies this test can be re-enabled on 32-bit MSVC again.